index.js

```diff
@@ -1,20 +1,524 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+const { SlowBuffer } = require("buffer");
+const readline = require("readline");
+const readlineInterface = readline.createInterface(
+  process.stdin,
+  process.stdout
+);
 
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     readlineInterface.question(questionText, resolve);
   });
 }
 
-start();
+/* Each room Object has:
```

Comment:
You could use this comment as the basis of a Class constructor next time to save some typing/time